web-ui/src/main/resources/catalog/views/default/templates/recordView/metadata.html

```diff
@@ -21,11 +21,9 @@ <h3 data-translate="">updatedOn</h3>
   ></span>
 </div>
 
-<div
-  data-ng-init="mdLanguages = [mdView.current.record.mainLanguage].concat(mdView.current.record.otherLanguage || [])"
-  data-aang-if="mdLanguages.length > 0"
-  class="gn-margin-bottom flex-row"
->
+<div data-ng-init="mdLanguages = getMdLanguages()"
+     data-aang-if="mdLanguages.length > 0"
+     class="gn-margin-bottom flex-row">
```

Comment:
Maybe

```suggestion
<div
  data-ng-init="mdLanguages = [mdView.current.record.mainLanguage].concat(mdView.current.record.otherLanguage || []).unique()"
  data-ng-show="mdLanguages.length > 0"
  class="gn-margin-bottom flex-row"
>
```

to also not displayed this section if language not declared ?

Reply:
> to also not displayed this section if language not declared ?

What do you mean by "the language is not declared", are you refering to `mdView.current.record.mainLanguage` ?

@fxprunayre what is the difference between `unique()` and the `getMdLanguages()` ?
If `mdView.current.record.mainLanguage` is undefined, the array will still be `[undefined]` with a `length > 0`.

Also, as `unique` is not an official part of `Array.prototype`, I would not use it, it's confusing to rely on behaviors that are not standard, but anyway, it's not a big deal.